### ISSUE TITLE
description in `SubParser`s

### ISSUE THREAD

**Issue by kopp** [typed-argparse/typed-argparse#33]
Is it possible to specify some kind of description of the sub-command?

For example in [the example](https://typed-argparse.github.io/typed-argparse/high_level_api/) I would like to tell the user what `<program> foo start` will do, what `<program> foo stop` will do etc.

I searched for a parameter to `tap.SubParser` like `description` but found none.  Other "natural" choices where to place such a description were (in my opinion)
- the docstring of the business logic functions (e.g. [`run_foo_start` here](https://typed-argparse.github.io/typed-argparse/high_level_api/) ) (as click does it)
- the docstring of the argument class derived from `TypedArgs` (e.g. [`FooStartArgs` here](https://typed-argparse.github.io/typed-argparse/high_level_api/) )

**Comment by bluenote10** [comment]
Yes, this is a frequently requested feature, just requested in the lasted opened issue #32 as well.

I've just added some minimal support in #34 which adds:
- `help` in subparsers
- `description` in subparser groups

That's basically what argparse allows. I hope this resolves the issue?

New version 0.2.8 is available now on PyPI.
